packages/core/src/animation/Animator.ts

```diff
@@ -124,21 +141,40 @@ export class Animator extends Component {
   crossFade(
     stateName: string,
     normalizedTransitionDuration: number,
+    layerIndex?: number,
+    normalizedTimeOffset?: number
+  ): void;
+
+  crossFade(transition: AnimatorStateTransition, layerIndex?: number): void;
```

Comment:
`This does not make any sense for the usability of the public API.` you can warp a internal method